src/sonic-eventd/rsyslog_plugin/syslog_parser.h

```diff
@@ -0,0 +1,26 @@
+#ifndef SYSLOG_PARSER_H
+#define SYSLOG_PARSER_H
+
+#include <vector>
+#include <string>
+#include <regex>
+#include "json.hpp"
+#include "events.h"
+
+using namespace std;
+using json = nlohmann::json;
```

Comment:
do not use usings in header files

Reply:
Same as above.

Yet, if you insist as style of use, we could move to cpp code. Your call.